files/build_templates/init_cfg.json.j2

```diff
@@ -29,7 +29,7 @@
                    ("teamd", "enabled", "enabled")] %}
 {%- if include_iccpd == "y" %}{% do features.append(("iccpd", "disabled", "enabled")) %}{% endif %}
 {%- if include_mgmt_framework == "y" %}{% do features.append(("mgmt-framework", "enabled", "enabled")) %}{% endif %}
-{%- if include_nat == "y" %}{% do features.append(("nat", "disabled", "enabled")) %}{% endif %}
+{%- if include_nat == "y" %}{% do features.append(("nat", "enabled", "enabled")) %}{% endif %}
```

Comment:
nat is an optional feature, it should remind the disable state until user explicitly enable this feature.

Reply:
Thanks @lguohan, these optional features can be enabled using 'sudo config feature state nat enabled'??

Reply:
yes